src/translation/translation_pl_PL.ts

```diff
@@ -3330,42 +3330,42 @@ Twój kanał został wyciszony i włączono „Wycisz mnie”. Napraw przyczynę
     <message>
         <location filename="../serverdlg.cpp" line="56"/>
         <source>Directory Type combo box</source>
-        <translation type="unfinished"></translation>
+        <translation>Lista wyboru typu serwera</translation>
     </message>
     <message>
         <location filename="../serverdlg.cpp" line="60"/>
         <source>Directory</source>
-        <translation type="unfinished">Serwer</translation>
+        <translation>Serwer</translation>
     </message>
     <message>
         <location filename="../serverdlg.cpp" line="61"/>
         <source>Select &apos;%1&apos; not to register your server with a directory.</source>
-        <translation type="unfinished"></translation>
+        <translation>Zaznacz „%1” żeby nie rejestrować twojego serwera w katalogu zbiorczym.</translation>
```

Comment:
Not sure about the quoting. Probably it's ok like that?

Reply:
We were speaking about the quoting long time back - it is.

Reply:
Ok. Good to know.